frontend/.env.example

```
@@ -9,10 +9,17 @@ APP_URL=
 
-NEXT_PUBLIC_STRAPI_API_URL=http://localhost:1337
-STRAPI_ACCESS_TOKEN= #empty if using docker compose
+STRAPI_API_URL=
+STRAPI_ACCESS_TOKEN=
 
-NEXT_PUBLIC_APP_ENV=test
\ No newline at end of file
+AWS_CDN_URL=
+AWS_S3_BUCKET_NAME=
+AWS_S3_BUCKET_ROOT=
+AWS_S3_BUCKET_REGION=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The AWS_S3_BUCKET_REGION key should go before the AWS_S3_BUCKET_ROOT key

(UnorderedKey)